### PR TITLE
Adding Commit After Commit Guideline

### PR DESCRIPTION
**Pull request by chamisfum**
# Description

Make a Commit After Commit Guideline

## Task reference

Initial Task to help the tech team in their task.

## Type of change

> Create Commit After Commit Guideline :
> [x] Clear definition
> [x] Provide a relevant picture
> [x] Provide a relevant test

# How has this been tested?

- [x] Expected results
- [x] Result screenshot

### **Before editing:**
   
   You can see that my last commit message is `create How to review a pull request from the member at first run markdown file`. I want to change this message to be more relevant to my last change about the `Commit After Commit Guideline`.

   alt
   
   The commit message in my pull request
   
   alt

  ### **After editing:**
   
   Here I changed my last commit message to `Adding Commit After Commit Guideline` as seen below.

   alt

   The commit message in my pull request after editing
   
   alt

# Checklist:

- [x] My changes do not affect any errors in existing features
- [x] Perform `git pull --rebase origin main` before ready to merge
- [x] No conflict in my changes
- [x] Any conflicts are resolved
- [x] I've follows [the code style guidelines](https://)
- [x] I've follows [the code documentation guidelines](https://)
- [x] I've added and run a test scenario that proves my changes are works
- [x] I've provided  all relevant screenshots
- [x] My changes generate no new warnings and no error
- [x] Approve by at least  2 or more tech member
- [x]  Approve by the project owner
- [x] Using all relevant labels

### CODE REVIEW

**Review by DSC-UMM — APPROVED** [review]
Good to go

File: Guideline/FIRST_RUN.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
+# First time developer guideline
```
Comment (ChamdaniMust) [inline Guideline/FIRST_RUN.md:1]: Why did you move the guide line file since there is only one file in the folder?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ Please include a summary of the changes and the related issue. Please also inclu
 
 ## Task reference
 
-Please include task reference link and details (name, requirement, etc) here.
+Please include the task reference link and details (name, requirement, etc) here.
 
-> **Example :** Create login page for student. [Task Reference](https://put_task_reference_link_here)
+> **Example:** Create a login page for the student. [Task Reference](https://put_task_reference_link_here)
```
[the remark this answers is not in the document]
Comment (ChamdaniMust) [reply .github/pull_request_template.md:9]: I see, it is okay then

File: Guideline/FIRST_RUN.md (hunk 1 above)
Comment (chamisfum) [reply Guideline/FIRST_RUN.md:1]: I want to make the guidelines better by separating them according to the function of each guideline to make it easier to access and read in the next task.

**Review by ChamdaniMust — APPROVED** [review]
